src/main/kotlin/org/jetbrains/research/testspark/bundles/llm/LLMDefaultsBundle.kt

```diff
@@ -13,5 +13,5 @@ object LLMDefaultsBundle : DynamicBundle(LLMBundlePaths.defaults) {
      * Gets the requested default value.
      */
     @Nls
-    fun get(@PropertyKey(resourceBundle = LLMBundlePaths.defaults) key: String): String = getMessage(key)
+    fun get(@PropertyKey(resourceBundle = LLMBundlePaths.defaults) key: String): String = getMessage(key).split("|").last()
```

Comment:
This one should be commented; not trivial my splitting by "|" is needed.

Reply:
in the new version the Dynamic bundle returns the whole path + the value at the end. and each is separated by `|`. For example: `"|b|properties.llm.LLMDefaults|k|maxLLMRequest|3"` without this the plugin verification etc, will throw:
` java.lang.NumberFormatException: For input string: "|b|properties.llm.LLMDefaults|k|maxLLMRequest|3"`

Reply:
Added a comment explaining the purpose of this split